xbmc/addons/AddonRepos.h

```diff
@@ -21,6 +21,15 @@ class CAddonMgr;
 class CRepository;
 class IAddon;
 
+/**
+ * Struct - AddonUpdate
+ */
+struct AddonUpdate
```

Comment:
Think we need for it a bit other name. 
By a view on `std::map<std::string, AddonUpdate>` it first looks that defined to e.g. a `enum`.

Maybe to use as `CAddonUpdate`, maybe also name a bit more detailed :thinking: 

@phunkyfish a idea about another name?

Reply:
how about `COutdatedWithUpdate` ? (also a bit confusing...)

Reply:
It could be more generic. It could be any installed addon with an update.

So maybe `CAddonWithUpdate`.

One argument is `m_installed‘ and the other is the `m_update`.